proto/vulcanize/auction/v1beta1/tx.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+package vulcanize.auction.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "vulcanize/auction/v1beta1/types.proto";
+
+option go_package = "github.com/tharsis/ethermint/x/auction/types";
+
+// MsgCreateAuction defines a create auction message
+message MsgCreateAuction {
+    option (gogoproto.goproto_getters) = false;
+    
+    // Duration of the commits phase in seconds
+    int64 commits_duration = 1;
```

Comment:
Can you use `timestamp.duration` instead of int64 
https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration